src/pow.cpp

```diff
@@ -18,7 +18,7 @@ unsigned int GetNextWorkRequired(const CBlockIndex* pindexLast, const CBlockHead
     if (pindexLast == NULL)
         return nProofOfWorkLimit;
 
-    if (params.fPowAllowMinDifficultyBlocks && pindexLast->nHeight >= 277299)
+    if (params.fPowAllowMinDifficultyBlocks && pindexLast->nHeight >= 277299 && pindexLast->nHeight < 1100000)
```

Comment:
Move both fork heights to chainparams

Reply:
They only apply to testnet (because that's the only one with fPowAllowMinDifficultyBlocks enabled). Still think it's worth it?

Reply:
Yes, all forks should be documented/handled there.  The params in the other networks could be anything (i.e. 0 or 1 or whatever) since it should never apply.

Reply:
Fixed.